<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Battery tester</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabs">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab1">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QWidget" name="plot_placeholder" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>400</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="en_checkbox">
            <property name="text">
             <string>Enabled</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="ranges">
            <property name="title">
             <string>Settings</string>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <property name="labelAlignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Voltage</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Current</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="set_current">
               <property name="minimumSize">
                <size>
                 <width>90</width>
                 <height>0</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>90</width>
                 <height>0</height>
                </size>
               </property>
               <property name="suffix">
                <string> A</string>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="stepType">
                <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Timer</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QTimeEdit" name="set_timer">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>90</width>
                 <height>0</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>90</width>
                 <height>0</height>
                </size>
               </property>
               <property name="currentSection">
                <enum>QDateTimeEdit::HourSection</enum>
               </property>
               <property name="displayFormat">
                <string>h:mm:ss</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Label</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="lineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>90</width>
                 <height>0</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>90</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>B1</string>
               </property>
               <property name="maxLength">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="set_voltage">
               <property name="minimumSize">
                <size>
                 <width>90</width>
                 <height>0</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>90</width>
                 <height>0</height>
                </size>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="suffix">
                <string> V</string>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="stepType">
                <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Readings</string>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Voltage</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="readVoltage">
               <property name="palette">
                <palette>
                 <active>
                  <colorrole role="WindowText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>85</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </active>
                 <inactive>
                  <colorrole role="WindowText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>85</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </inactive>
                 <disabled>
                  <colorrole role="WindowText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>160</red>
                     <green>162</green>
                     <blue>162</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </disabled>
                </palette>
               </property>
               <property name="font">
                <font>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="text">
                <string>0.0 V</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Current</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="readCurrent">
               <property name="palette">
                <palette>
                 <active>
                  <colorrole role="WindowText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>170</red>
                     <green>0</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </active>
                 <inactive>
                  <colorrole role="WindowText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>170</red>
                     <green>0</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </inactive>
                 <disabled>
                  <colorrole role="WindowText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>160</red>
                     <green>162</green>
                     <blue>162</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </disabled>
                </palette>
               </property>
               <property name="font">
                <font>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="text">
                <string>0.0 A</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="readCapAH">
               <property name="font">
                <font>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="text">
                <string>0.0 AH</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Capacity</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Capacity</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="readCapWH">
               <property name="font">
                <font>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="text">
                <string>0.0 WH</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="readTime">
               <property name="font">
                <font>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="text">
                <string>00:00:00</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Time</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="resetButton">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
